tests/product/test_server_uninstall.py

```diff
@@ -108,54 +78,8 @@ def test_uninstall_lost_host(self):
         cmd_output = self.run_prestoadmin('server uninstall',
                                           raise_error=False)
         self.assertRegexpMatches(cmd_output, expected)
-        process_per_active_host = []
-        for host, pid in process_per_host:
-            if host not in down_node:
-                process_per_active_host.append((host, pid))
-        self.assert_stopped(process_per_active_host)
 
         for container in [self.cluster.internal_master,
                           self.cluster.internal_slaves[1],
                           self.cluster.internal_slaves[2]]:
             self.assert_uninstalled_dirs_removed(container)
-
-    def test_uninstall_with_dir_readonly(self):
```

Comment:
That's not what it tests.  it tests that the remote user can't uninstall if it doesn't have sudo privileges.  that's going to continue to be true.